src/explore-education-statistics-common/src/modules/release/components/NationalStatisticsSection.tsx

```diff
@@ -9,20 +9,25 @@ export default function NationalStatisticsSection({
     <>
       {showHeading && <h3>National statistics</h3>}
       <p>
-        The{' '}
-        <a href="https://www.statisticsauthority.gov.uk/">
-          United Kingdom Statistics Authority
-        </a>{' '}
-        designated these statistics as National Statistics in accordance with
+        These accredited official statistics have been independently reviewed by
         the{' '}
+        <a href="https://osr.statisticsauthority.gov.uk/what-we-do/">
+          Office for Statistics Regulation
+        </a>{' '}
+        (OSR). They comply with the standards of trustworthiness, quality and
+        value in the{' '}
+        <a href="https://code.statisticsauthority.gov.uk/the-code/">
+          Code of Practice for Statistics
+        </a>
+        . Accredited official statistics are called National Statistics in the{' '}
         <a href="https://www.legislation.gov.uk/ukpga/2007/18/contents">
           Statistics and Registration Service Act 2007
-        </a>{' '}
-        and signifying compliance with the Code of Practice for Statistics.
+        </a>
+        .
       </p>
       <p>
-        Designation signifying their compliance with the authority's{' '}
-        <a href="https://www.statisticsauthority.gov.uk/code-of-practice/the-code/">
+        Accreditation signifies their compliance with the authority's{' '}
+        <a href="https://code.statisticsauthority.gov.uk/the-code/">
```

Comment:
I think the link here is slightly different to the one in the ticket but I may be wrong?
In the ticket description that Cam sent me it had this link `https://www.statisticsauthority.gov.uk/code-of-practice/the-code/`

Reply:
... Which looks like the original link actually! So maybe it was a mistake in the ticket and your change is correct? Might be worth checking but I think the chances are you had it right!

Reply:
cool :)